tls/s2n_ktls.c

```diff
@@ -126,6 +224,9 @@ static S2N_RESULT s2n_ktls_configure_socket(struct s2n_connection *conn, s2n_ktl
  */
 int s2n_connection_ktls_enable_send(struct s2n_connection *conn)
 {
+    /* gate this feature only to tests until it is ready for release */
+    POSIX_ENSURE(s2n_in_test(), S2N_ERR_NOT_IN_TEST);
+
```

Comment:
This shouldn't be necessary since you haven't added this method to any api file yet

Reply:
I think we should keep this since this is now a destructive operation (offload to kTLS). Its a good sanity measure and will also prevent the lib from calling it itself.